app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendRegistrationStartView.scala.html

```diff
@@ -22,22 +22,23 @@
         table: GovukTable
 )
 
-@(eclReference: String)(implicit request: Request[_], messages: Messages)
-
+@(eclReference: Option[String])(implicit request: Request[_], messages: Messages)
 
 @layout(
     pageTitle = title(messages("start.amend.title")),
     timeout = false,
     showBackLink = false,
-    showSignOut = false
+    showSignOut = false,
+    registrationTypeForTopBanner = Some(Amendment),
+    eclReferenceForStartLink = eclReference
 ) {
 
     <h1 class="govuk-heading-l">@messages("start.amend.h1")</h1>
 
     @table(Table(
         rows = Seq(
             Seq(TableRow(content = Text(messages("start.amend.table.registrationNumber")), classes = "govuk-summary-list__key"),
-                TableRow(content = Text(eclReference), classes = "govuk-body"))
+                TableRow(content = Text(eclReference.get), classes = "govuk-body"))
```

Comment:
We need to handle when the value is null because `.get` will throw an exception if eclReference is empty. You could now display the row if it's null or if eclReference being null means something has gone horribly wrong with the service (in that case, why is it an option in the first place) then we need to handle that in the controller and return a 500 there?